vendor/mitt/README.md

```diff
@@ -177,3 +179,5 @@ Pull requests are the greatest contributions, so be sure they are focused in sco
 ## License
 
 [MIT License](https://opensource.org/licenses/MIT) © [Jason Miller](https://jasonformat.com/)
+
+<!-- prettier-ignore-end -->
```

Comment:
Let's not change vendored subdirectories. Instead let's exclude `vendor/*` from the script (perhaps using [`.prettierignore`](https://prettier.io/docs/en/ignore.html#ignoring-files-prettierignore)?)

Reply:
Thanks, indeed much nicer suggestion I've introduced the ignore file in https://github.com/puppeteer/puppeteer/pull/7014/commits/1f394174d366ff0654f9a4f699591785412a3f08.